### PR TITLE
Update DragAction.js

### PR DESCRIPTION
**Pull request by belfarz**
Fixes #1086  (<=== Add issue number here)
link to original issue ==>https://github.com/publiclab/Leaflet.DistortableImage/blob/main/src/edit/actions/DragAction.js

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: src/edit/actions/DragAction.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ L.DragAction = L.EditAction.extend({
   },
 
   addHooks: function() {
-    var edit = this._overlay.editing;
+    let edit = this._overlay.editing;
```
Comment (PeculiarE) [inline src/edit/actions/DragAction.js:16]: Use the 'const' keyword instead.

**Review by PeculiarE — CHANGES_REQUESTED** [review]

**Comment by PeculiarE** [comment]
One more thing, please link the PR to the original issue. Currently, you're linking the PR to itself.

**Comment by jywarren** [comment]
Hi @PeculiarE thank you so much for helping out! Thanks @belfarz for your contribution!

**Comment by belfarz** [comment]
hello, Thanks for the corrections but were can i get the link to the
original issue im kinda lost and cant find it

**Comment by belfarz** [comment]
> Hi @PeculiarE thank you so much for helping out! Thanks @belfarz for your contribution!

**Review by PeculiarE — APPROVED** [review]
Well done! @belfarz...this looks ready for merging.

@TildaDares....please assist to merge. Thank you!

**Pull request merged**